app/models/case/sar/offender.rb

```diff
@@ -1,33 +1,72 @@
-class Case::SAR::Offender < Case::SAR::Standard
+class Case::SAR::Offender < Case::Base
   class << self
     def type_abbreviation
       'OFFENDER'
```

Comment:
Can this be changed to `OFFENDER_SAR`

Reply:
I've had a go at this and it breaks things; obviously I'm not changing it in all the right places...! But I see you're changing this in your PR and I think we should not change it here therefore